i18nextBrowserLanguageDetector.js

```diff
@@ -162,6 +162,23 @@
     }
   };
 
+  var url = {
+    name: 'url',
+
+    lookup: function lookup(options) {
+      var found = void 0;
+      if (typeof window !== 'undefined') {
+        var language = window.location.pathname.match(/\/([a-zA-Z-]*)/g);
```

Comment:
if we would lookup in `window.location.href` we could also handle urls like en.mydomain.com / de.mydomain.com

Reply:
wouldn't that be hostname? which would eventual better be done in another lookup? @terhuerne opinions?

Reply:
I think it would be more straight forward for a user if we could have an additional lookup called `subdomain` because this could be confusing for the user as we would count against different parts of the URL. But it would be little effort to add the subdomain lookup as well. I'll commit something in the next minutes. ;-)

Reply:
@terhuerne agree makes sense to separate